faction.cpp

```diff
@@ -54,9 +53,19 @@ char const *tp[] = {
 	"long",
 	"map"
 };
-
 char const **TemplateStrs = tp;
 
+// Quit states
+const string qs[] = {
+	"none",
+	"quit order",
+	"quit by gm"
+	"quit and restart",
+	"won game"
+	"game over",
+};
+const string *QuitStrs = qs;
```

Comment:
Needed these so I could output a textual string for the various quit states in the faction json.